src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing/01 instrumentHTTP.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'instrumentHTTP'
+excerpt: 'instrumentHTTP instruments the k6 http module with tracing capabilities.'
+---
+
+The `instrumentHTTP` function instruments the k6 http module with tracing capabilities. It transparently replaces each of the k6 http module functions with versions that automatically attach a trace context to every request. Functions that you can instrument include [del](/javascript-api/k6-http/del), [get](/javascript-api/k6-http/get), [head](/javascript-api/k6-http/head), [options](/javascript-api/k6-http/options), [patch](/javascript-api/k6-http/patch), [post](/javascript-api/k6-http/post), [put](/javascript-api/k6-http/head), and [request](/javascript-api/k6-http/request).
+
+The `instrumentHTTP` automatically adds tracing information to HTTP requests.
+This means that, to instrument the HTTP requests, you don't need to rewrite any code.
+Instead, call it once in the init context.
+From that point forward, all requests made by the HTTP module from that point forward will have a trace context header added to them, and the metadata for the data-point output will have the used `trace_id`. For details about propagation, refer to [About trace contexts](/javascript-api/k6-experimental/tracing#about-trace-contexts).
+
+The `instrumentHTTP` function takes a single configuration object argument that can be used to configure the tracer. The argument should be an instance of the [`Options`](/javascript-api/k6-experimental/tracing/options) object.
```

Comment:
`instance of` makes it sound like `Options` is some kind of type while it is just a description of fields that are recognised by `instrumentHTTP` and `Client`

I see that for `k6/http` we mostly [just link](https://k6.io/docs/javascript-api/k6-http/request/) to it and do not (at least) use the wording "instance of". 

Can we maybe move to the same format - were we have "Parameter" and "returns" tables?

Reply:
I actually forgot to make a table here 🙇 Switching to the same format as http indeed 👍